<?xml version='1.0' ?>
<launch>
  <arg name='x_min' default='-10.0' />
  <arg name='x_max' default='10.0' />
  <arg name='y_min' default='-10.0' />
  <arg name='y_max' default='10.0' />
  <arg name='z_min' default='2.5' />
  <arg name='z_max' default='10.0' />

  <node name='uav_demo' pkg='control' type='uav_demo' output='screen'>
    <param name='x_min' value='$(arg x_min)' />
    <param name='x_max' value='$(arg x_max)' />
    <param name='y_min' value='$(arg y_min)' />
    <param name='y_max' value='$(arg y_max)' />
    <param name='z_min' value='$(arg z_min)' />
    <param name='z_max' value='$(arg z_max)' />
  </node>
</launch>
